src/capi.rs

```diff
@@ -444,6 +444,26 @@ pub unsafe extern fn mp4parse_get_track_video_info(parser: *mut mp4parse_parser,
     MP4PARSE_OK
 }
 
+#[no_mangle]
+pub unsafe extern fn mp4parse_is_fragmented(parser: *mut mp4parse_parser, track_id: u32, fragmented: *mut bool) -> mp4parse_error {
```

Comment:
The extern api is C, not C++ so it is safer to use `*mut u8` instead of `*mut bool` for the out-parameter here. I'm sure it works, because rust-llvm and C stdbool will likely end up with the same representation, but this isn't guaranteed.